### DISCUSSION TITLE
用Omit处理DatePickerProps后部分属性不能被识别

### DISCUSSION THREAD

**Discussion by GuanJdoJ** [ant-design/ant-design#41486] [Q&A]
antd5中，DatePickerProps中有showTime属性
我自定义了个属性用于重写DatePickerProps的value属性
代码示例如下
```
import React from 'react';
import { DatePickerProps } from 'antd';

type PropsType = Omit<DatePickerProps, 'value'> & {
  value?: string; 
};

const MyComp: React.FC<PropsType> => {...}

export default  MyComp;
```
```
//此时会报错，不存在showTime
< MyComp showTime/>
```

此时在用PropsType的组件中再使用showTime就会报TS类型错误，请问下大家这个问题怎么解决

**Comment by yoyo837** [comment]
上demo

**Reply by GuanJdoJ** [reply]
https://codesandbox.io/s/omittun-datepickerpropsbu-fen-shu-xing-6u7v84?file=/src/App.js

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/30386473/227929483-9212cdc1-9443-4ede-bcfe-9b0d4465056e.png">

**Comment by GuanJdoJ** [comment]
#41487
